content/shared/translations/bios_de.yaml

```diff
@@ -379,7 +379,7 @@ conway:
   born: 1937
   died: 2020
   bio: |
-    __John Horton Conway__ (geb. 1937) ist ein britischer Mathematiker und derzeit
+    __John Horton Conway__ (1937 - 2020) ist ein britischer Mathematiker und derzeit
```

Comment:
John Conway is still alive – we used to have "2020" as the "died" property for all currently living mathematicians, but that is no longer needed in the latest version.

Reply:
👍